tailwind.config.ts

```diff
@@ -13,6 +13,15 @@ const config: Config = {
         'gradient-conic':
           'conic-gradient(from 180deg at 50% 50%, var(--tw-gradient-stops))',
       },
+      colors: {
+        awhite: '#ffffff',
+        ablack: '#363636',
+        agrey: '#999999',
+        adarkgrey: '#3f4042',
+        ared: '#c3352e',
+        adeepred: '#ac362e',
+        ablue: '#0275d8 '
+      }
```

Comment:
**Question:** Overwriting the tailwind presets: is it something we cannot do, or something we do not want to do?

Reply:
We can do it but I thought it would be best to keep the vast array of color presets tailwind has to offer is they may come in useful:

https://tailwindcss.com/docs/customizing-colors